ansible/roles/devspace/handlers/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: restart devspace
+  environment:
+    USER_ID: "{{ user_id }}"
+  become: yes
+  become_user: "{{ devuser }}"
+  include: devspace-runtime.yml
```

Comment:
Do you think name of the file could become a var? I have more complex playbook as an example of managing containers https://github.com/aleksandra-tarkowska/infrastructure/blob/extend_devspace/ansible/devspace-extend.yml